src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,64 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Shooter extends SubsystemBase {
+  /**
+   * Creates a new Shooter.
+   */
+  private CANSparkMax shooter = new CANSparkMax(9, MotorType.kBrushless);
+  private Solenoid adjustmentSolenoid = new Solenoid(5);
+  private Solenoid rotationSolenoid = new Solenoid(6);
+
+  public Shooter() {
+
+  }
```

Comment:
Depending on what the design shows, may need to specify 2 motor controllers and for one to follow the other